easybuild/easyconfigs/b/Bison/Bison-3.0.4-GCCcore-6.4.0.eb

```diff
@@ -19,10 +23,9 @@ builddependencies = [
     ('binutils', '2.28', '', True),
 ]
 
-
 sanity_check_paths = {
+    'dirs': ['bin', 'lib'],
     'files': ['bin/%s' % x for x in ['bison', 'yacc']] + ['lib/liby.a'],
-    'dirs': [],
```

Comment:
We consistently include `files` before `dirs` in all other easyconfigs, so we should stick to that?

I do understand the motivation for this (alphabetical sorting), but it's better to be consistent with existing easyconfigs?

In addition, it's rather silly to check for non-empty `bin` & `lib` directories if we're also checking for files in those subdirectories?